index.bs

```diff
@@ -2636,12 +2636,14 @@ Classes of Changes</h4>
 		<dt id=class-1>
 			1. No changes to text content
 		<dd>
-			These changes include fixing broken links, style sheets or invalid markup.
+			These changes include fixing broken links, style sheets, or invalid markup.
 
 		<dt id=class-2>
-			2. Corrections that do not affect conformance
+			2. Changes that do not functionally affect interpretation of the document
```

Comment:
That sentence seems to be missing a word. Did you mean "Changes that do not functionally **but** affect interpretation of the document"?

Reply:
No, no word missing. "These changes do not affect interpretation of the document (in a functional sense, as opposed to legibility)"